integration-webcert/src/main/java/se/inera/intyg/minaintyg/integration/webcert/converter/data/value/ValueToolkit.java

```diff
@@ -0,0 +1,62 @@
+package se.inera.intyg.minaintyg.integration.webcert.converter.data.value;
+
+import java.util.List;
+import se.inera.intyg.minaintyg.integration.api.certificate.model.CertificateQuestion;
+import se.inera.intyg.minaintyg.integration.api.certificate.model.value.CertificateQuestionValueText;
+import se.inera.intyg.minaintyg.integration.webcert.client.dto.config.CertificateDataConfig;
+import se.inera.intyg.minaintyg.integration.webcert.client.dto.config.CertificateDataConfigCheckboxMultipleDate;
+import se.inera.intyg.minaintyg.integration.webcert.client.dto.config.CheckboxMultipleDate;
+import se.inera.intyg.minaintyg.integration.webcert.client.dto.value.CertificateDataValue;
+import se.inera.intyg.minaintyg.integration.webcert.client.dto.value.CertificateDataValueDate;
+import se.inera.intyg.minaintyg.integration.webcert.client.dto.value.CertificateDataValueDateList;
+
+public final class ValueToolkit {
+
+  private static final String NOT_PROVIDED = "Ej angivet";
+
+  private ValueToolkit() {
+    throw new IllegalStateException("Utility class");
+  }
+
+  public static List<CertificateQuestion> getValueDateListSubQuestions(
```

Comment:
Will this be extracted into its own converter like for the other value types?

Reply:
Yes, as we discussed we will replace this with a new converter that doesn't create subquestions, but instead a question value type that can cater for the need.